tensorflow/g3doc/tutorials/deep_cnn/index.md

```diff
@@ -50,7 +50,7 @@ that systematically decrements over time.
 for input
 data to isolate the model from disk latency and expensive image pre-processing.
 
-We also provide a multi-GPU version of the model which demonstrates:
+We also provide a [multi-GPU version](#multi-GPU) of the model which demonstrates:
```

Comment:
Can you use the anchor tag that is automatically generated (which also works on github): `#training-a-model-using-multiple-gpu-cards`